OpenRA.Mods.Common/LoadScreens/BlankLoadScreen.cs

```diff
@@ -77,7 +77,13 @@ public void StartGame(Arguments args)
 			// Load a replay directly
 			if (!string.IsNullOrEmpty(Launch.Replay))
 			{
-				var replayMeta = ReplayMetadata.Read(Launch.Replay);
+				ReplayMetadata replayMeta = null;
+				try
+				{
+					replayMeta = ReplayMetadata.Read(Launch.Replay);
+				}
+				catch { }
+
 				if (ReplayUtils.PromptConfirmReplayCompatibility(replayMeta, Game.LoadShellMap))
```

Comment:
Hmm. If `ReplayMetadata.Read` throws, `replayMeta` will be left with a `null` value, right?  If so, `ReplayUtils.PromptConfirmReplayCompatibility` will produce an "Outdated engine" error message, which is misleading.